perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -1343,49 +1496,9 @@ void MapBasedPredictionNode::updateObjectData(TrackedObject & object)
   return;
 }
 
-void MapBasedPredictionNode::removeOldObjectsHistory(
-  const double current_time, const TrackedObjects::ConstSharedPtr in_objects)
+void MapBasedPredictionNode::cleanupOldStoppedOnGreenTimes(
```

Comment:
What does this name mean??

Reply:
Not the best agree, but it actually cleanups the dictionary with such name. I’ll think of a better name